src/tools/nvmf-rbd-gw/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+#
+# We postpone and build libspdk_bdev_rbd library from here and not
+# from SPDK in order to avoid complicated dependencies: bdev_rbd.c
+# depends on librbd and librados include paths as well.
+#
+include(ExternalProject)
+ExternalProject_Add(spdk_bdev_rbd
+    SOURCE_DIR ${CMAKE_SOURCE_DIR}/src/spdk/module/bdev/rbd
+    BUILD_IN_SOURCE 1
+    BUILD_COMMAND env CFLAGS=-I${CMAKE_SOURCE_DIR}/src/include V=1 $(MAKE)
+    BUILD_ALWAYS 1
+    DOWNLOAD_COMMAND ""
+    UPDATE_COMMAND ""
+    CONFIGURE_COMMAND ""
+    INSTALL_COMMAND "")
+unset(make_cmd)
+
+add_library(libspdk_bdev_rbd STATIC IMPORTED)
+set_property(TARGET libspdk_bdev_rbd PROPERTY IMPORTED_LOCATION ${CMAKE_SOURCE_DIR}/src/spdk/build/lib/libspdk_bdev_rbd.a)
+add_dependencies(libspdk_bdev_rbd librados librbd spdk_bdev_rbd)
+
+set(nvmf_rbd_gw_src nvmf-rbd-gw.c)
+add_executable(nvmf-rbd-gw ${nvmf_rbd_gw_src})
```

Comment:
Nit: create a rpm/deb package for this new tool

Reply:
Yep, good point, will do